dbt_worker/tasks.py

```diff
@@ -103,9 +128,53 @@ def _insert_log_path(command: List[str], task_id: str):
         return
     command.insert(0, LOG_PATH_ARGS)
     command.insert(1, get_task_artifacts_path(task_id, None))
+    command.insert(2, LOG_FORMAT_ARGS)
```

Comment:
I'm just curious do you prefer let IDE set in request explicitly? Or have a hard coded code here. 

I tends to not make any unnecessary hard coded logic in worker because it's implicit and hard to debug. Because log format is relatively trivial so I'm neutral on this. Let me know you final decision.

Reply:
I think we'd prefer to set it here. Dbt CLI defaults to `text` format, and it doesn't make a lot of sense for `dbt-server` to use that default if we're also setting a `--log-path` for a default (the `text` format isn't easily read from a log file). We also don't want the IDE to have to specify the format on every request. I think if in the end the `dbt-server` is actually used for multiple cases, we could revisit and possibly set with an env var, but for the time being I think this is a sensible default.